### PR TITLE
Matrix URL

### PR DESCRIPTION
**Pull request by stappersg**
Incomplete merge request that asks for how to connect through Matrix.

### CODE REVIEW

**Comment by stappersg** [comment]
image

**Comment by joshka** [comment]
We're just testing this right now using a free bridge bot (https://t2bot.io/discord/), and didn't want to announce it as the main place to chat just yet until we've "kicked the tires" for a little bit. We haven't bridged all the channels either.

The space is at https://matrix.to/#/#ratatui:matrix.org though, feel free to come chat there. There's a bit of lag sometimes between discord and the matrix channels and threads don't seem to be handled well.

**Review by joshka — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -140,10 +141,12 @@ the community forked the project and created this crate. We look forward to cont
...
-feel free to join and come chat! There are also plans to implement a [Matrix](https://matrix.org/)
-bridge in the near future. **Discord is not a MUST to contribute**. We follow a pretty standard
-github centered open source workflow keeping the most important conversations on GitHub, open an
-issue or PR and it will be addressed. 😄
+feel free to join and come chat! There is also a [Matrix](https://matrix.org/) bridge available at
+[#ratatui:matrix.org](https://matrix.to/#/#ratatui:matrix.org).
+
+**Discord is not a MUST to contribute**. We follow a pretty standard github centered open source
+workflow keeping the most important conversations on GitHub, open an issue or PR and it will be
+addressed. 😄
```
Comment (kdheepak) [inline README.md:149]: What do you think about rephrasing this slightly?

```suggestion
While we do utilize Discord for coordinating, it's not essential for contributing. Our primary open-source workflow is centered around GitHub. For significant discussions, we rely on GitHub — please open an issue, a discussion or a PR. 
```

Comment (joshka) [reply README.md]: LGTM - will squash

**Review by orhun — APPROVED** [review]

**Pull request merged**